qa/multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/40_scroll.yml

```diff
@@ -0,0 +1,40 @@
+---
+"Scroll on the mixed cluster":
+
+  - do:
+      search:
+        index: my_remote_cluster:test_index
```

Comment:
maybe also scroll against local indices at the same time? That would complicate the test a bit though, maybe it's not worth it, not sure.